templates/islandora-large-image.tpl.php

```diff
@@ -33,7 +33,7 @@
     <?php endif; ?>
   </div>
   <div class="islandora-large-image-metadata">
-    <?php print $description; ?>
+    <?php // print $description; ?>
```

Comment:
While i am OK merging this, i will be adding my own pull to remove it later since commented out code is a coder violation. The untouched template will still be in the large image solution pack.